DateTimeCalendar/DateTime.java

```diff
@@ -0,0 +1,159 @@
+public class DateTime{
+    private Date date;
+    private Time time;
+    
+    private DateTime(){
+
+    }
+
+//Constructors
+    public DateTime(int year, Date.Month month, int day, int hour, int minute, int second) {
+        setDate(year, month, day);
+        setTime(hour, minute, second);
+    }
+
+    public DateTime (Date date, Time time) {
+        setDate(date);
+        setTime(time);
+    }
+
+    public DateTime(int year, Date.Month month, int day) {
+        setDate(year, month, day);
+    }
+
+    public DateTime(int hour, int minute, int second) {
+        setTime(hour, minute, second);
+    }
+
+    public DateTime(Date date) {
+        setDate(date);
+    }
+
+    public DateTime(Time time) {
+        setTime(time);
+    }
+//
+
+//Methods
+    public DateTime add(int years, int months, int days, int hours, int minutes, int seconds) {
+        if (years >= 0 && months >= 0 && days >= 0 && hours >= 0 && minutes >= 0 && seconds >= 0) {
+            int combindSeconds = time.getSecond(true) + (hours * 3600 + minutes * 60 + seconds);
+            int additionalDays = 0;
+    
+            if ( combindSeconds >= 86400) {
+                //Obtain full "Days" from the total amount of seconds
+                additionalDays += combindSeconds / 86400;
```

Comment:
`86400` is a magic number, I'd suggest using a constant for this

Reply:
Corrected this by making `private static final int SEC_IN_DAY = 86400;`